.eslintrc.js

```diff
@@ -1,139 +1,143 @@
 module.exports = {
```

Comment:
Looks like this file got auto-formatted. Happy to revert if easier to review

Reply:
@G2Jose Yep. Can you revert it? Thanks

Reply:
Done!